numba/cpython/builtins.py

```diff
@@ -57,8 +57,36 @@ def generic_is(context, builder, sig, args):
         return cgutils.false_bit
 
 
+@lower_builtin(operator.is_, types.Boolean, types.BooleanLiteral)
+@lower_builtin(operator.is_, types.BooleanLiteral, types.Boolean)
+def boolean_literal_cross_type_is(context, builder, sig, args):
+    sig_arg0, arg1 = sig.args
+
+    if isinstance(sig_arg0, types.BooleanLiteral):
+        literal_sig_arg = sig_arg0
+        non_literal_arg = args[1]
+    else:
+        literal_sig_arg = arg1
+        non_literal_arg = args[0]
+
+    
+    # QUESTION: how do I call the already defined casting instead?
+    #           This is just copy-paste from lower_cast
+    lit = context.get_constant_generic(
```

Comment:
There's a `context.cast()` for this. See https://github.com/numba/numba/blob/fd8c232bb37a1945e8dc8becef9fe05fdd78c4cf/numba/core/base.py#L694

Reply:
Thanks. That simplified the code.